lib/shared/setup-tsci-packages.ts

```diff
@@ -5,7 +5,7 @@ import { execSync } from "node:child_process"
 
 export function setupTsciProject(
   directory = process.cwd(),
-  dependencies = ["@types/react", "@tscircuit/core"],
+  dependencies = ["@types/react", "@tscircuit/core", "@tscircuit/runframe"],
```

Comment:
this shouldn't be a dependency of the user's project. There should be a way for us to build the CLI such that it's not required.

We could have `@tscircuit/cli` as a devDependency which would technically solve this i guess

Reply:
alternatively, we can just prevent the error from printing- i think this is a good solution because it means the user can install a custom runframe version if they want to. In any case let's make sure we don't have anything we don't need as a dependency. Everything here should be a devDependency- i also just realized the variable name might be bad